PIP-1.md

```diff
@@ -0,0 +1,43 @@
+# PIP-1 Point-Chain common components
+
+## Motivation
+
+Point chain intended to have several ecosystem layer applications. Those application can reuse similar components. It can reduce amount of work and make applications better connected with each other.
+
+
+## Details
+
+There are 5 types of reusable components on Point network
+
+* Development tools 
+* Smart contracts
+* Reusable backend
+* Web UI 
+* Data
+
+### Reusable Smart Contracts
```

Comment:
* Not only libraries. It can be any type of common dependencies which apps can use/extend/derive etc.
* Yes. we can do it in a separate PR.
* We have discussed it already on the meeting, so you know disadvantages already.